README.md

```diff
@@ -49,3 +49,4 @@ This project uses the following third party crates:
 * mbox (MIT)
 * bitfield (MIT and Apache-2.0)
 * pkg-config (MIT and Apache-2.0)
+* enumfags2 (MIT OR Apache-2.0)
```

Comment:
I know that the repo/crate states that its license is MIT **or** Apache-2.0, but we decided to display it as **and** - the motivation being that since the license text allows both, contributors can choose either one, so technically the library will have both MIT and Apache-2.0 licensed code inside of it.

Reply:
This have now been fixed. Also fixed the incorrect spelling of the name.